### ISSUE TITLE
Inverse + Animated Mesh = Still fluid inflow.

### ISSUE THREAD

**Issue by Mitsuma** [rlguy/Blender-FLIP-Fluids-Beta#216]
A weird issue I found.
![blender_2018-03-20_15-07-44](https://user-images.githubusercontent.com/8568089/37659524-8493c764-2c50-11e8-81a6-cade22f29b6b.png)

Very basic scene setup, an inflow object inside a mesh which is set to "inverse".
Everything is fine when it is not set to also export the animated mesh.
![2018-03-20_14-49-35](https://user-images.githubusercontent.com/8568089/37659571-a23d0f14-2c50-11e8-9e4e-a694d087a908.png)
Not shown but it also collides just fine with the mesh as expected with Inverse.

But once I also check "Export Animated mesh", the fluid will stay where it is as shown in the first picture.
Both images are at the same frame, one is just with exported mesh and one without.

Edit: I actually just checked again and it seems like it does not like keyframes? 
Tried to bake it without animated mesh but it stays.
Even without keyframes and without exported animated mesh it is still stuck in place.

**Comment by Mitsuma** [comment]
I actually removed everything I changed and the fluid would still not move. 
Ended up deleting the inflow object and replaced it with a new one. 

Now it works again, at least without exporting animated mesh.
Which should be fine with just animated rotation anyways, forgot about that.

Even enabling animated mesh export works fine now.
So I have no clue why the fluid actually got stuck. Doesn't want to reproduce.
I have encountered this one time before actually, although in that case I scrapped the project.
Also saved over the blend file from the first post, so can't provide that.
